docs/generated/packages/react/generators/setup-ssr.json

```diff
@@ -39,6 +39,12 @@
         "hidden": true,
         "description": "Extra include entries in tsconfig.",
         "default": []
+      },
+      "bundler": {
+        "description": "The bundler to use.",
+        "type": "string",
+        "enum": ["rspack", "webpack"],
+        "default": "webpack"
```

Comment:
should this be Rspack?

Reply:
no, setup-ssr can still be used in non-module federation applications. only module-federation related generators have been updated to default to rspack